build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -5,11 +5,24 @@
 @fluent_bit_common_conf_path = "/etc/opt/microsoft/docker-cimprov/fluent-bit-common.conf"
 
 @os_type = ENV["OS_TYPE"]
+@isWindows = false
 if !@os_type.nil? && !@os_type.empty? && @os_type.strip.casecmp("windows") == 0
+  @isWindows = true
   @fluent_bit_conf_path = "/etc/fluent-bit/fluent-bit.conf"
   @fluent_bit_common_conf_path = "/etc/fluent-bit/fluent-bit-common.conf"
 end
 
+@using_aad_msi_auth = false
+if !ENV["USING_AAD_MSI_AUTH"].nil? && !ENV["USING_AAD_MSI_AUTH"].empty? && ENV["USING_AAD_MSI_AUTH"].strip.casecmp("true") == 0
+  @using_aad_msi_auth = true
+end
+
+@geneva_logs_integration = false
```

Comment:
This a big PR and it touching lot of scenarios, can we capture list of the scenarios which are being validated as part of this PR?

Reply:
Added scenarios in the description